frontend/src/components/WidgetContainerTitleGroup.js

```diff
@@ -22,6 +40,13 @@ const WidgetContainerTitleGroup = ({
         </div>
         {children}
       </div>
+      <div>
+        {
+        (menuItems && menuItems.length > 0 && (
```

Comment:
menuItems will never be falsy, you can remove the first part of this if statement

Reply:
Wasnt sure was more of a just in case. Will fix.